community-operators/okd-windows-machine-config-operator/0.0.1/okd-windows-machine-config-operator.0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,229 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    categories: "OpenShift Optional"
+    certified: "false"
+    containerImage: quay.io/openshift-windows/okd-windows-machine-config-operator:4.6-1d646e8
+    createdAt: 2020-09-01T11:59:59Z
+    description: An operator which allows for Windows workloads to be run within OKD
+    operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/suggested-namespace: windows-machine-config-operator
+    repository: https://github.com/openshift/windows-machine-config-operator
+    support: ""
+  name: okd-windows-machine-config-operator.v0.0.1
+  namespace: windows-machine-config-operator
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions: {}
+  description: |-
+    # OKD Windows Machine Config Operator
+
+    ## WARNING
+    Community distribution of the Windows Machine Config Operator. This is an alpha build, and is not meant for
+    production workloads. Issues with this distribution of WMCO are not supported.
```

Comment:
>Issues with this distribution of WMCO are not supported.

I suggest a softer wording :-) Link to the WMCO GitHub issues page and say something to the effect of "best effort".